zipline/algorithm.py

```diff
@@ -1366,25 +1367,22 @@ def order_value(self,
                     limit_price=None,
                     stop_price=None,
                     style=None):
-        """Place an order by desired value rather than desired number of
-        shares.
+        """
+        Place an order for a fixed amount of value.
```

Comment:
"amount of value" sounds a little weird. Maybe just "value"?

Reply:
Changed this to "amount of money".